projects/ngx-uploader/src/lib/ngx-uploader.class.ts

```diff
@@ -43,9 +46,10 @@ export class NgUploaderService {
   }
 
   handleFiles(incomingFiles: FileList): void {
+    console.error('handleFiles incomingFiles', incomingFiles);
```

Comment:
Remove console.error. No customer will look at the console to understand why the file was rejected.

Reply:
Oh, yeah, my mistake. Fixed